scripts/docker-build.sh

```diff
@@ -64,6 +64,7 @@ fi
 if [[ "X${APP_PACKAGE_NAME}" != "Xqfield" ]]; then
   grep "ch.opengis.qfield" -l -r ${SOURCE_DIR}/android/ | xargs sed -i "s/ch.opengis.qfield/ch.opengis.${APP_PACKAGE_NAME}/g"
   grep "ch\.opengis\.qfield" -l -r ${SOURCE_DIR}/src/ | xargs sed -i "s/ch\.opengis\.qfield/ch.opengis.${APP_PACKAGE_NAME}/g"
+  grep "ch\/opengis\/qfield" -l -r ${SOURCE_DIR}/src/ | xargs sed -i "s/ch\/opengis\/qfield/ch\/opengis\/${APP_PACKAGE_NAME}/g"
```

Comment:
```suggestion
  grep "ch\/opengis\/qfield" -l -r ${SOURCE_DIR}/src/ | xargs sed -i "s|ch/opengis/qfield|ch/opengis/${APP_PACKAGE_NAME}/g"
```
slightly better readability